tests/client/test_linalg.py

```diff
@@ -132,8 +131,15 @@ def testSVD(self):
         cxt = tc.Context()
         cxt.matrix = tc.tensor.Dense.load((m, n), tc.F32, matrix.flatten().tolist())
         cxt.result = tc.linalg.svd(cxt.matrix)
-
-        self.assertRaises(tc.error.NotImplemented, lambda: self.host.post(ENDPOINT, cxt))
+        cxt.U_shape = cxt.result['U'].shape
```

Comment:
you can access the shape of a returned tensor like `response['U'][0][0]`, that way you don't have to make three separate requests